_pages/openid-connect.md

```diff
@@ -239,38 +241,83 @@ grant_type=authorization_code
   "access_token": "hhJES3wcgjI55jzjBvZpNQ",
   "token_type": "Bearer",
   "expires_in": 3600,
-  "id_token": "eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiJ9.eyJpc3MiOiJodHRwczovL2xvZ2luLmdvdiIsImF1ZCI6InVybjpnb3Y6Z3NhOm9wZW5pZGNvbm5lY3Q6ZGV2ZWxvcG1lbnQiLCJzdWIiOiI5ZTQ5NTUzZC04OTRiLTRlNTMtOTE3YS0yOTc4MjRmZGY5NmYiLCJhY3IiOiJodHRwOi8vaWRtYW5hZ2VtZW50Lmdvdi9ucy9hc3N1cmFuY2UvbG9hLzMiLCJub25jZSI6bnVsbCwianRpIjoiYzBiOGQ5Mzg2NDgzMzI3MzQyNzI3ZTdkNzUwYmM2YTEiLCJleHAiOjE0ODY0Nzg1NTYsImlhdCI6MTQ4NjQ3ODU1OCwibmJmIjoxNDg2NDc4NTU4fQ.UK0plb_sf14ql2ibl7wm7JH8IhfTH9V6wJIjMLIqkZPvtAaAsOugLm73moGl3WnJIQ35_zQeSpu-MiVR0pq0fYjMo7GHTYTd2FeVCdxzCZQHOqFIQ0ydX69ekts73Toe1qNFbFXu_tt2JGJEF5miKd5r5WteGT5ERZI8R23XQT2Y-nzqZgO8HOAbvyR3EeCtD3WY7GqGmWL00xwKh-YwYkr_j5BF44yhbpVVwaSgjH8YIxyUEtf_oc7P5XpEICRLzT3A2WDkXF0yAKGmq9PDS9_7wEqfarKgnZTs4nMNWkE19Oj9Xku55jrBYun4u85PKGCr-16gigmEvethfB2p0Q"
+  "id_token": "eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiJ9.eyJzdWIiOiJiMmQyZDExNS0xZDdlLTQ1NzktYjlkNi1mOGU4NGY0ZjU2Y2EiLCJpc3MiOiJodHRwczovL2lkcC5pbnQubG9naW4uZ292IiwiYWNyIjoiaHR0cDovL2lkbWFuYWdlbWVudC5nb3YvbnMvYXNzdXJhbmNlL2xvYS8xIiwibm9uY2UiOiJhYWQwYWE5NjljMTU2YjJkZmE2ODVmODg1ZmFjNzA4MyIsImF1ZCI6InVybjpnb3Y6Z3NhOm9wZW5pZGNvbm5lY3Q6ZGV2ZWxvcG1lbnQiLCJqdGkiOiJqQzdOblU4ZE5OVjVsaXNRQm0xanRBIiwiYXRfaGFzaCI6InRsTmJpcXIxTHIyWWNOUkdqendsSWciLCJjX2hhc2giOiJoWGpxN2tPcnRRS196YV82dE9OeGN3IiwiZXhwIjoxNDg5Njk0MTk2LCJpYXQiOjE0ODk2OTQxOTgsIm5iZiI6MTQ4OTY5NDE5OH0.pVbPF-2LJSG1fE9thn27PwmDlNdlc3mEm7fFxb8ZADdRvYmDMnDPuZ3TGHl0ttK78H8NH7rBpH85LZzRNtCcWjS7QcycXHMn00Cuq_Bpbn7NRdf3ktxkBrpqyzIArLezVJJVXn2EeykXMvzlO-fJ7CaDUaJMqkDhKOK6caRYePBLbZJFl0Ri25bqXugguAYTyX9HACaxMNFtQOwmUCVVr6WYL1AMV5WmaswZtdE8POxYdhzwj777rkgSg555GoBDZy3MetapbT0csSWqVJ13skWTXBRrOiQQ70wzHAu_3ktBDXNoLx4kG1fr1BiMEbHjKsHs14X8LCBcIMdt49hIZg"
 }
 ```
 
- * **access_token**
+ * **access_token** (string)
     An opaque token used to authenticate to the [User Information](#user-information) endpoint
 
- * **token_type**
+ * **token_type** (string)
     Describes the kind of access token. Will always be `Bearer`.
 
- * **expires_in**
+ * **expires_in** (number)
```

Comment:
I am saying it will work, and have confirmed it with the JWT gem in Ruby:

```ruby
past = JWT.encode({ exp: Time.now.to_f - 10 }, nil, 'none')
=> "eyJ0eXAiOiJKV1QiLCJhbGciOiJub25lIn0.eyJleHAiOjE0ODk3NzY3MjUuMjI1MTUyfQ."
JWT.decode(past, nil, false)
=> JWT::ExpiredSignature: Signature has expired


future = JWT.encode({ exp: Time.now.to_f + 10 }, nil, 'none')
=> "eyJ0eXAiOiJKV1QiLCJhbGciOiJub25lIn0.eyJleHAiOjE0ODk3NzY3NzkuODUwMzAzfQ."
JWT.decode(future, nil, false)
=> [{"exp"=>1489776779.850303}, {"typ"=>"JWT", "alg"=>"none"}]
```